### ISSUE TITLE
Minor mistakes in trainer.py

### ISSUE THREAD

**Issue by hzykent** [HuguesTHOMAS/KPConv#50]
Hi @HuguesTHOMAS,

There seem to be two minor mistakes in the trainer.py file.
 
- ` 164 for ign_label in model.config.ignored_label_inds:`
      `165              ignored_bool = tf.logical_or(ignored_bool, model.labels == ign_label)`
This line should be changed to the way you wrote in other parts of your code.
`  for ign_label in model.config.ignored_label_inds:
                    ignored_bool = tf.logical_or(ignored_bool, tf.equal(model.labels, ign_label))`
The original line will simply output all zeros.

- `182 self.prob_logits = tf.nn.softmax(new_logits)`
This line should be changed to the way you wrote in tester.py.
`self.prob_logits = tf.nn.softmax(model.logits)`
Otherwise it will be collected by wrong indices in this step.
` 696            # Eliminate shadow indices
                    b = b[b < max_ind-0.5]`
                    `# Get prediction (only for the concerned parts)
                    probs = stacked_probs[b]
                    inds = point_inds[b]
                    c_i = cloud_inds[b_i]`

The reason that the original code didn't complain here is that the ignored_bool was all zero.

Thank you again for sharing your work.

**Comment by HuguesTHOMAS** [comment]
Hi @hzykent,

Thank you for your detailed explanation. I think you are right, but cannot check the code right now. I will do it in the week adn correct the mistakes.

Best,
Hugues
